scalafix-tests/unit/src/test/scala/scalafix/tests/rule/MavenFuzzSuite.scala

```diff
@@ -26,7 +26,7 @@ class MavenFuzzSuite extends FunSuite with DiffAssertions {
       sourceJars: Seq[Path],
       tmp: Path
   ): Seq[Path] = {
```

Comment:
I think this diff would have fixed the compile error as well
```suggestion
  ): collection.Seq[Path] = {
```